### PR TITLE
[Site] Lighten bootstrap

### PR DESCRIPTION
**Pull request by smnandre**
Remove unused components

To be honest, i have something a bit more "radical" in mind ... but let's go there slowly :)

### CODE REVIEW

File: ux.symfony.com/assets/styles/app.scss [hunk 1]
```diff
@@ -26,16 +26,16 @@
 @import "../../vendor/twbs/bootstrap/scss/tables";
 @import "../../vendor/twbs/bootstrap/scss/forms";
 @import "../../vendor/twbs/bootstrap/scss/buttons";
-@import "../../vendor/twbs/bootstrap/scss/transitions";
-@import "../../vendor/twbs/bootstrap/scss/dropdown";
- @import "../../vendor/twbs/bootstrap/scss/button-group";
+// @import "../../vendor/twbs/bootstrap/scss/transitions";
```
Comment (kbond) [inline ux.symfony.com/assets/styles/app.scss:29]: Why commented instead of removed?

Comment (smnandre) [reply ux.symfony.com/assets/styles/app.scss:29]: To keep track of what's in the standard distribution VS what we "removed" 

It's scss so it won't appear in compiled code, but i can clean this if you want, i know what the next step is ... and it won't require this file   👼

Comment (smnandre) [reply ux.symfony.com/assets/styles/app.scss:29]: (but don't know _when_)

Comment (kbond) [reply ux.symfony.com/assets/styles/app.scss:29]: Oh I see, understood!

**Comment by kbond** [comment]
Thanks Simon.

**Pull request merged**